draft-ietf-acme-acme.md

```diff
@@ -1944,11 +1932,13 @@ for the domain in question.
 
 The path at which the resource is provisioned is comprised of the fixed prefix
 ".well-known/acme-challenge/", followed by the "token" value in the challenge.
-The value of the resource MUST be the ASCII representation of the key
```

Comment:
This is also a change to normative language, so let's keep it in for now. Also, I'm not sure why this can be deleted. Is it redundant with somewhere else?